### PR TITLE
Add default quickstart support for Chinese LaTex support.

### PR DESCRIPTION
**Pull request by cdarlint**
Most common use of Chinese LaTeX is xelatex+ctex with encoding UTF-8, and there are many other possible configurations for different tasks.
For example, another option is to use pdflatex+ctex with encoding GBK.
Generally no one uses basic "latex" command to generate chinese pdfs as it is outdated, unmaintained and too complicated to configure.

### CODE REVIEW

**Comment by cdarlint** [comment]
I've considered adding this configuration to writers/latex.py.
There are two main reason I put it in quickstart:
1 default latex-engine after quickstart does not compile chinese documents, but babel or other language options in writers/latex.py locates after latex-engine cofiguration, which does not apply to chinese documents
2 not every chinese documents needs to be compiled by xelatex+ctex (many other options available), it should be configurable by end-user as in conf.py

**Comment by tk0miya** [comment]
Thank you for proposal.

Now sphinx also provides default settings to Japanese.
https://github.com/sphinx-doc/sphinx/blob/1.5.1/sphinx/builders/latex.py#L275

I believe it would be nice if we can merge your settings into `setup()` function in latex builder.
I will refactor it to add default settings for specific languages in this weekend.
So please wait for that.

**Comment by tk0miya** [comment]
BTW, Is this change stable?
We should not merge this into stable branch if this brings breaking changes.
I don't know about chinese docs much, but I'm afraid that the new default settings affect to chinese users and docs.

**Comment by cdarlint** [comment]
before this change it is not possible to generate chinese pdf by default.
this change only has affect to latex output with language start with "zh_", no affect on other output format/language
after this change it is possible to generate a tex file with chinese characters that can compiles with xelatex

There are some blogs talking about workarounds on sphinx chinese pdf generation, they alter the output tex file manually.
I'd appreciate if it generate a tex that compiles without manual modification.

Thank you for your reply, I'll check builders/latex.py recently, which I didn't read before.
I'm working on a chinese document and I'll follow up this issue.

**Comment by cdarlint** [comment]
for chinese latex output,
some people modify generated .tex file, while some other people modify conf.py to generate a .tex that compiles
I propose the quickstart.py to generate a conf.py that works for zh_* languages by default

File: sphinx/templates/quickstart/conf.py_t
Comment (tk0miya) [inline sphinx/templates/quickstart/conf.py_t:144]: Jinja2 raises TemplateSyntaxError in this line:
```
TemplateSyntaxError: expected token 'end of statement block', got '.'
```
https://travis-ci.org/sphinx-doc/sphinx/jobs/184147701#L2019

Does this really work correct in your environment?

File: sphinx/templates/quickstart/conf.py_t
Comment (tk0miya) [inline sphinx/templates/quickstart/conf.py_t:149]: `usepackage` is an internal variable, so we can't say this always works fine.
It might be overwritten by any extensions. (It is used for `Sphinx.add_latex_package() API.)

How about this? I believe it might work fine.
```
latex_elements = {'polyglossia': '\\usepackage{ctex}'}
```

**Review by tk0miya — CHANGES_REQUESTED** [review]
As I commented above, I prefer to provide the default settings in `setup()` function in LaTeX builder.
I think `conf.py` should be used for "customization". We should not add a default settings to the file.

So please update this PR based on #3251.

File: sphinx/templates/quickstart/conf.py_t
Comment (cdarlint) [reply sphinx/templates/quickstart/conf.py_t:144]: this runs correct in my environment, but I haven't touched tox before and I can not configure it to work correctly for now
I run this program successfully, on windows anaconda python3

File: sphinx/templates/quickstart/conf.py_t
Comment (cdarlint) [reply sphinx/templates/quickstart/conf.py_t:149]: base on this version, it does not work. Because it set 'multilingual' and 'babel' to empty string on 'ja' only.

So 'zh_*' would have a multilingual property and got an error on method "babel_renewcommand".
This method generate tex file lines like:
```\addto\captionsenglish{\renewcommand{\figurename}{图 }}```
This is not correct.

I believe after your refactor last weekend, it will be easier to find a place to add chinese support.
I'll check the latest version later, thanks.

**Comment by tk0miya** [comment]
Now we start to discuss in new PR #3272.
So I close this.

Thanks,

**Pull request closed without merging**
